code/modules/admin/tabs/round_tab.dm

```diff
@@ -165,12 +165,13 @@
 	set name = "Start Round"
 	set desc = "Start the round RIGHT NOW"
 	set category = "Server.Round"
-
-	if (!SSticker)
-		alert("Unable to start the game as it is not set up.")
-		return
 	if (alert("Are you sure you want to start the round early?",,"Yes","No") != "Yes")
 		return
+	if (SSticker.current_state == GAME_STATE_STARTUP)
+		alert("Game is setting up and will launch as soon as it is ready.")
+		message_admins(SPAN_BLUE("[usr.key] has started the process to start the game when loading is finished."))
+		while (SSticker.current_state == GAME_STATE_STARTUP)
```

Comment:
alternatively, you could register a signal to COMSIG_GLOB_MODE_PREGAME_LOBBY which fires when the ticker moves to GAME_STATE_PREGAME to then SSticker.request_start

but you don't really gotta

Reply:
I am not going to do this unless I have to because I am now committed to make the "DON'T USE WHILE" people mad